addons/event/static/src/views/event_registration_kanban_controller.js

```diff
@@ -0,0 +1,43 @@
+import { kanbanView } from "@web/views/kanban/kanban_view";
+import { KanbanController } from "@web/views/kanban/kanban_controller";
+import { EventRegistrationSummaryDialog } from "@event/client_action/event_registration_summary_dialog";
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+
+export default class EventRegistrationKanbanController extends KanbanController {
+
+    setup() {
+        super.setup()
+        this.dialog = useService("dialog");
+        this.orm = useService("orm");
+    }
+
+    async openRecord(record, mode) {
+        if (this.props.context.is_registration_desk_view) {
+            const barcode = record.data.barcode;
+            const eventId = record.data.event_id[0];
```

Comment:
Can we create records in the views opened from registration_view? Just to be sure event_id[0] won't crash trying to access something not there.

Reply:
Can you please elaborate I didn't got your point.

PS: Event (`event_id`) is mandatory field in the attendee form view.

Reply:
I was thinking editable records, but after checking it should be ok.